wordpress-mobile/WordPress-iOS
# Feature: Add missing settings storage and network layer

**First step for** #7831

In this PR I'm adding storage and network support for the following settings:

- Date Format
- Time Format
- Start of the Week
- Posts per page
- AMP Supported (read only in the apps)
- AMP Enabled

**To test:**

There's no much to test here without a UI, so you can check that the tests pass and that the code looks good. If you would rather review this once the UI is in place I'm happy to add that to this PR, but I thought it might be easier to split it. Your call @koke
